app/models/health/eligibility_inquiry.rb

```diff
@@ -22,8 +22,7 @@ class EligibilityInquiry < HealthBase
     has_many :batches, class_name: 'Health::EligibilityInquiry', foreign_key: :batch_id, dependent: :destroy
 
     scope :pending, -> do
-      where.not(inquiry: nil).
-        where.not(id: Health::EligibilityResponse.select(:eligibility_inquiry_id))
+      where.not(inquiry: nil).where.missing(:eligibility_response)
```

Comment:
Where is the `missing` scope coming from?

Reply:
`where.missing` is a nifty Rails 6.1 feature: https://apidock.com/rails/v6.1.7.7/ActiveRecord/QueryMethods/WhereChain/missing